GoogleForCreators/web-stories-wp
# Redesign: Implemented dropdowns

## Context

Implements dropdowns in the new design in the editor.

## User-facing changes

| Before | After |
|-|-|
| ![Screen Shot 2021-03-10 at 20 23 01](https://user-images.githubusercontent.com/637548/110721472-a374dd80-81de-11eb-930a-1bf2f57eee4d.png) | ![Screen Shot 2021-03-10 at 20 22 05](https://user-images.githubusercontent.com/637548/110721474-a5d73780-81de-11eb-8046-9b28d807a6fa.png) |
| ![Screen Shot 2021-03-10 at 20 23 06](https://user-images.githubusercontent.com/637548/110721483-a8399180-81de-11eb-9cdc-4770edec5829.png) | ![Screen Shot 2021-03-10 at 20 22 14](https://user-images.githubusercontent.com/637548/110721485-aa9beb80-81de-11eb-8148-22250f7bf88c.png) | 
| ![Screen Shot 2021-03-08 at 00 54 39](https://user-images.githubusercontent.com/637548/110283501-311db680-7fae-11eb-9939-e5b460de45fb.png) | ![Screen Shot 2021-03-08 at 00 53 55](https://user-images.githubusercontent.com/637548/110283506-33801080-7fae-11eb-913d-822a966a5bd0.png) |
| ![Screen Shot 2021-03-08 at 00 54 44](https://user-images.githubusercontent.com/637548/110283511-35e26a80-7fae-11eb-976a-7ba608c8623a.png) | ![Screen Shot 2021-03-08 at 00 54 00](https://user-images.githubusercontent.com/637548/110283516-3844c480-7fae-11eb-8ea2-3bfed38fe86a.png) |
| ![Screen Shot 2021-03-08 at 01 28 24](https://user-images.githubusercontent.com/637548/110283525-3a0e8800-7fae-11eb-9df8-ad25d9f5170f.png) |  ![Screen Shot 2021-03-08 at 01 27 37](https://user-images.githubusercontent.com/637548/110283530-3c70e200-7fae-11eb-9e37-abc9ae326369.png) |

## Testing Instructions

### QA

This PR can be tested by following these steps:

1. Filter media elements
2. Hover a media element to open the submenu
3. Edit the color of any element and see the dropdown for local or global colors

### UAT




## Checklist




---

Fixes #6644
